app/src/main/res/layout/fragment_edit_about.xml

```diff
@@ -0,0 +1,123 @@
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="16dp"
+            android:fontFamily="sans-serif"
+            android:text="@string/edit_about_short_description"
+            android:textColor="@color/middleGrey"
+            android:textSize="12sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <EditText
+            android:id="@+id/etShortDescription"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="16dp"
+            android:background="@drawable/rounded_grey_border_white_background"
+            android:ellipsize="end"
+            android:includeFontPadding="true"
+            android:paddingLeft="5dp"
+            android:paddingTop="4dp"
+            android:paddingRight="5dp"
+            android:paddingBottom="4dp"
+            android:fontFamily="@font/open_sans"
+            android:hint="@string/public_profile_about_hint"
+            android:maxLength="200"
+            android:textColor="@color/middleGrey"
+            android:textSize="16sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="1.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/tvShortDescriptionTitle" />
+
+        <TextView
+            android:id="@+id/tvLongDescriptionTitle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginEnd="16dp"
+            android:fontFamily="sans-serif"
+            android:text="@string/edit_about_long_description"
+            android:textColor="@color/middleGrey"
+            android:textSize="12sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/etShortDescription" />
+
+        <EditText
+            android:id="@+id/etLongDescriptionText"
+            android:layout_width="0dp"
+            android:layout_height="208dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="16dp"
+            android:background="@drawable/rounded_grey_border_white_background"
+            android:fontFamily="@font/open_sans"
+            android:gravity="fill_horizontal"
+            android:hint="@string/edit_about_long_description_hint"
+            android:includeFontPadding="true"
+            android:paddingLeft="5dp"
```

Comment:
we should use the newer padding convetions, paddingStart and paddingEnd. Or use vertical padding